### PR TITLE
fix: convert to string from null in hs

### PR DESCRIPTION
**Pull request by aashishmalik**
## What are the changes introduced in this PR?

convert to string from null in hs

## What is the related Linear task?

Resolves INT-1531

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3136/test-report.html

**Review by mihir-4116 — APPROVED** [review]

File: src/v0/destinations/hs/util.js [hunk 1]
```diff
@@ -223,7 +224,8 @@ const getTransformedJSON = async (message, destination, propertyMap) => {
       // lowercase and replace ' ' & '.' with '_'
       const hsSupportedKey = formatKey(traitsKey);
       if (!rawPayload[traitsKey] && propertyMap[hsSupportedKey]) {
-        let propValue = traits[traitsKey];
+        // HS accepts empty string to remove the property from contact
```
Comment (utsabc) [inline src/v0/destinations/hs/util.js:227]: Can you add some inline HS docs for this?

File: test/integrations/destinations/hs/processor/data.ts [hunk 2]
```diff
@@ -5269,4 +5269,101 @@ export const data = [
...
+  {
+    name: 'hs',
+    description: 'Test coversion of null to string values',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
```
Comment (utsabc) [inline test/integrations/destinations/hs/processor/data.ts:5281]: Use payload builder for generating this paylaod check `testUtils.ts`

File: src/v0/destinations/hs/util.js (hunk 1 above)
Comment (aashishmalik) [reply src/v0/destinations/hs/util.js:227]: added in corresponding linear card

Comment (utsabc) [reply src/v0/destinations/hs/util.js:227]: No I am saying add link to Hs docs in your code for future ref

File: test/integrations/destinations/hs/processor/data.ts (hunk 2 above)
Comment (aashishmalik) [reply test/integrations/destinations/hs/processor/data.ts:5323]: done , in output couldn't use function as there are extra parameters for HS

**Review by mihir-4116 — APPROVED** [review]

**Pull request merged**